File: leet_code/binary_search/268/268.py
from typing import List


class Solution:
    def missingNumber(self,nums:List[int])-> int:
        n = len(nums)
        total = n * (n+1)//2
        current_sum = sum(nums)
        return total-current_sum

s = Solution()
print(s.missingNumber([0,1]))

File: leet_code/binary_search/268/268.dart
class Solution {
  int missingNumber(List<int> nums) {
    int n = nums.length;
    int total = n * (n + 1) ~/ 2;
    int currentSum = nums.fold(0, (sum, num) => sum + num);
    return total - currentSum;
  }
}

void main() {
  Solution s = Solution();
  print(s.missingNumber([0, 1, 3]));
}
